CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Change log
 -----------
 
+* Drop the external initramfs and only use the initramfs bundled in the kernel [Florin]
```

Comment:
@floion but the meta-resin update means a new release, right? Because the Jenkins builds will trigger a new deployment to staging with this new version you added in VERSION.